extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -149,6 +149,19 @@ private function getRelatedPages( $articleId, $limit = 6 ){
 		}
 	}
 
+	private function getCuratedContentSections() {
+		try {
+			$response = $this->sendRequest( 'CuratedContent', 'getList' )->getData();
+			if ( !empty( $response['sections'] ) ) {
+				return $response['sections'];
```

Comment:
We need to make sure what is the first scope of our iteration. I assumed we just want to display a list of sections and items at the beginning. I'll reach out to our PO to be 100% sure.